.changeset/silly-hairs-explain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@ithaka/pharos': patch
+---
+
+- Pharos button added to checkbox group. combobox, radio group and select in storybook
+- width modified for combobox and select in storybook
```

Comment:
```suggestion
modify button and input styles in Storybook:

* add `<pharos-button> to checkbox group, combobox, radio group and select stories
* modify width for combobox and select in Storybook
```
We use the [imperative mood](https://grammar.collinsdictionary.com/us/easy-learning/the-imperative) for our changelog entries